src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -31,0 +1,79 @@
+import { useState } from 'react';
+import { Filter } from '../../types/EnumFilter';
+import { FiltersType } from '../../types/Filter';
+
+type Props = {
+  getChangeFilters: (filter: FiltersType) => void;
+  filters: FiltersType;
+};
+
+export const TodoFilter: React.FC<Props> = ({ getChangeFilters, filters }) => {
+  const [clearInput, setClearInput] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+
+  const handleFiltersChange = (optionValue: Filter, inputQuery: string) => {
+    setInputValue(inputQuery);
+    setClearInput(true);
+    getChangeFilters({
+      option: optionValue as Filter,
+      filteredQuery: inputQuery,
+    });
+
+    if (inputQuery === '') {
+      setClearInput(false);
+    }
+  };
+
+  const handleClearFilters = () => {
+    setInputValue('');
+    getChangeFilters({
+      option: filters.option as Filter,
```

Comment:
Type Assertion: Similar to the previous comment, ensure that `filters.option` is correctly typed to avoid unnecessary type assertions.